cuda/test/preconditioner/jacobi_kernels.cpp

```diff
@@ -325,6 +325,34 @@ TEST_F(Jacobi, CudaPreconditionerEquivalentToRefWithMPW)
 }
 
 
+TEST_F(Jacobi, CudaTransposedPreconditionerEquivalentToRefWithMPW)
+{
+    initialize_data({0, 11, 24, 33, 45, 55, 67, 70, 80, 92, 100}, {}, {}, 13,
+                    97, 99);
+
+    auto bj = bj_factory->generate(mtx);
+    auto d_bj = d_bj_factory->generate(mtx);
```

Comment:
I prefer generating `d_bj` on empty matrix. how do you think?

Reply:
Do you think this would make a difference? I was just lazy, as usual :laughing:

Reply:
only for less testing time? I do not consider it too seriously. I am fine with current implementation

Reply:
The tests are already incredibly fast (< 50ms each as far as I remember), so I wouldn't worry about it.